packages/typescript/README.md

```diff
@@ -242,8 +242,8 @@ export default {
   input: './main.ts',
   plugins: [
     typescript({ module: 'CommonJS' }),
-    commonjs({ extensions: ['.js', '.ts'] }), // the ".ts" extension is required
-  ],
+    commonjs({ extensions: ['.js', '.ts'] }) // the ".ts" extension is required
```

Comment:
It looks like the README got formatted unintentionally

Reply:
Is it necessary to change it back? I suppose I could add the README to `.prettierignore`, but these seem like reasonable changes and I'm not sure we'd want to permanently avoid formatting the README?

Reply:
Yes this needs to be reverted, because it's inconsistent with the formatting used by the rest of the project.

Reply:
Hmm. It's just inconsistent because the other docs haven't been reformated yet. The others would get updated as those projects are formatted again with the latest version of prettier and eslint config

I've reverted the changes to the README, but it required turning off linting of the README. That doesn't really seem like the right solution to me, but I'm not quite sure how else I would avoid it being reformatted.

Reply:
The linting in the docs doesn't match what's used in our code. For example, single-argument arrow functions have no parenthesis in your linted version. That indicates that your local checkout might not be set up correctly - for example your editor is running formatting separate from our prettier config.

Reply:
Taking a closer look it seems like we're use a different config for the README? @shellscape can you weigh in on what format we should be using?

Reply:
I was able to drop the change to the README from this PR. I opened https://github.com/rollup/plugins/pull/676 with more details about this issue